imu_complementary_filter/CMakeLists.txt

```diff
@@ -38,11 +41,11 @@ ament_target_dependencies(complementary_filter geometry_msgs rclcpp message_filt
 
 install(TARGETS
         complementary_filter
-        DESTINATION lib/${PROJECT_NAME}
+        DESTINATION lib
 )
 install(TARGETS
         complementary_filter_node
-        DESTINATION lib/${PROJECT_NAME}
+        DESTINATION lib
```

Comment:
I believe the nodes should go into the subdirectory. At least `ros2 pkg create` also has `lib/${PROJECT_NAME}` here, so I'd leave it as it is.